src/test/java/core/basesyntax/MyLinkedListTest.java

```diff
@@ -350,44 +350,40 @@ public void removeObjectValueByItemPositive() {
         boolean isSecondCatRemove = cats.remove(THE_SAME_SECOND_CAT);
         Assert.assertTrue("Test failed! Result after removing should be true", isSecondCatRemove);
         Assert.assertEquals("Expected size is incorrect. Expected: 6, but was: " + cats.size(), 6, cats.size());
-        Assert.assertEquals("Unexpected element found in list on 0 position. Expected: " + FIRST_CAT +", but was: " + cats.get(0),
+        Assert.assertEquals("Unexpected element found in list on 0 position. Expected: " + FIRST_CAT + ", but was: " + cats.get(0),
```

Comment:
I'm not shure it's good idea to change tests file. Maybe it'll be better to back changes out

Reply:
i used hot key`s on my code, and it automatically changed code style

Reply:
nevertheless - you should not do it:)

no need to format the code, which is not in scope of your PR